chain-impl-mockchain/tests/common/arbitrary/address.rs

```diff
@@ -0,0 +1,38 @@
+use crate::common::address::AddressData;
+use chain_addr::{Discrimination, KindType};
+use quickcheck::{Arbitrary, Gen};
+
+#[derive(Clone, Debug)]
+pub struct ArbitraryAddressKind(pub KindType);
+
+impl Arbitrary for ArbitraryAddressKind {
+    fn arbitrary<G: Gen>(g: &mut G) -> Self {
+        match u8::arbitrary(g) % 3 {
+            0 => ArbitraryAddressKind(KindType::Single),
+            1 => ArbitraryAddressKind(KindType::Group),
+            2 => ArbitraryAddressKind(KindType::Account),
+            _ => unreachable!(),
+        }
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct ArbitraryAddressesData(pub Vec<AddressData>);
```

Comment:
This name seems like it will have a greppability problem when someone searches for `AddressData`.